vhd/lib/vhd-util-resize.c

```diff
@@ -207,6 +207,7 @@ swap(vhd_block_t *list, int a, int b)
 	list[b] = tmp;
 }
 
+#if 0
 static int
```

Comment:
Why no just delete it?

Reply:
Because there's a lot of code here and we might want to revisit whether it can be made to work safely. Deleting it make it more difficult to see that there is some history here and maybe a starting point.

Normally I'd agree on deleting unused code but I think this justifies being a special case..